ray-operator/config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,39 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
All the YAML below was either generated by kubebuilder or cargo-culted from a fresh kubebuilder-generated repo that had a lot of original boilerplate for webhooks and certmanager preserved.

Reply:
yes, this page https://book.kubebuilder.io/cronjob-tutorial/running-webhook#deploy-webhooks